robocopkaka/bootcamp-29-project
# #157507851 Paginate centers on frontend

#### What does this PR do?
paginate centers on frontend
#### Description of Task to be completed?
* install query-string
* install react-materialize
* use the Pagination component from react-materialize
* add a meta object to the center's object in intial state
* add a change page method that gets the appropriate centers for each page
* use the page number in the query params to send a request to fetch centers
* modify the url in the api method for retrieving centers to make use of a page parameter in it's query
* modify the action and action creator responsible for retrieving centers to make use of a page value as an argument
* modify the reducer method responsible for getting all centers to make use of the new api response format for retrieving centers
* set meta information inside the reducer method
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install all dependecies
* run `npm run start:all` to start up the application
* go to `localhost:8080` on your browser
* click on `Signup`
* enter your details
* click on `Signup`
* request for admin access
* click on `Centers`
* click on the red `add` button at the bottom right corner
* enter the details of the center
* click on `Add Center`
* add at least ten more centers
* click on `Centers`
* you should see a pagination link at the bottom of the page
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?(if applicable)
#157507851
